### ISSUE TITLE
Swedish chars in path, when doing F5(reload), component does not get loaded

### ISSUE THREAD

**Issue by joelmandell** [vuejs/vue-router#1366]
### Version
2.5.1

### Reproduction link
[https://jsfiddle.net/dikatlon/0mqk9m11/4/](https://jsfiddle.net/dikatlon/0mqk9m11/4/)

### Steps to reproduce
Click the link "I did not survice swedish characters". It gets loaded fine. Press F5, component does not show up.
In order to reproduce this you need to clone from git repo, because refreshing a jsfiddle is not so good idea (cannot refresh the frame either, gets post error message).

Git repo:https://github.com/joelmandell/swede-vue-router

### What is expected?
Component should get loade when reloading page, but because of swedishs chars(åäö) it does not work. So it seems :(

### What is actually happening?
It does not show up. Tested in FF and Chrome, I do not have IE at this moment.

**Comment by posva** [comment]
Dup of #838 
TLDR: don't use Unicode characters in route paths because they're not officially supported

**Comment by joelmandell** [comment]
Oh I see. Will update my code

Den 26 apr. 2017 11:04 skrev "Eduardo San Martin Morote" <
notifications@github.com>:
